dsl/src/main/scala/com/crobox/clickhouse/dsl/column/Magnets.scala

```diff
@@ -120,7 +154,7 @@ trait Magnets {
       with StringSearchOps
       with EmptyNonEmptyCol[C]
 
-  implicit def stringColMagnetFromString[T <: String: QueryValue](s: T): StringColMagnet[String] =
+  implicit def stringColMagnetFromString[T <: String : QueryValue](s: T): StringColMagnet[String] =
```

Comment:
The space between String and QueryValue doesn't seem right. Looks like formatting issues